clang/include/clang/Sema/Sema.h

```diff
@@ -4453,9 +4453,9 @@ class Sema final : public SemaBase {
                                       SourceLocation *ArgLocation = nullptr);
 
   /// Determine if type T is a valid subject for a nonnull and similar
-  /// attributes. By default, we look through references (the behavior used by
-  /// nonnull), but if the second parameter is true, then we treat a reference
-  /// type as valid.
+  /// attributes. We skip dependence By default, we look through references
```

Comment:
again, we're not skipping dependence by default, we're ALWAYS skipping dependence.  We skip references (presumably by default).  I think the dependence behavior just needs its own sentence here.

Reply:
Yes, i think i was planning to make it like `We skip dependence and By default,`